app/main.py

```diff
@@ -2,0 +1,28 @@
+import os
+
+
+def move_file(command: str) -> None:
+    command = command.split(" ")
+    mv = command[0]
+    file_to_move = command[1]
+    destination = command[2].split("/")
+    if mv != "mv":
```

Comment:
It's a good practice to check if the command is correct. However, since the function name is 'move_file', it's assumed that the command is 'mv'. So, this check can be skipped.